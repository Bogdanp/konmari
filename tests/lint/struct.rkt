#lang racket/base

(require racket/string)

(provide
 user?
 user
 user-username
 user-password
 user-password-hash)

(struct user
  (username password-hash)
  #:transparent)

(define u (user "a" ""))
(set-user-username! u "b")
