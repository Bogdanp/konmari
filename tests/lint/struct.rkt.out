/struct.rkt:9:2:error:identifier 'user-password' provided but not defined
